app/views/teams/_form.html.erb

```diff
@@ -6,8 +6,8 @@
    </div> 
   </div>
   <% if params[:action] == 'edit' %>
-    <%= team.submit t('shared.update_resource', resource_name: t('shared.team')), class: 'btn btn-primary' %>	
+    <%= team.submit t('shared.update_resource', resource_name: t('shared.team')), class: 'btn btn-info' %>	
```

Comment:
Submit buttons should actions be in primary color. Update/Override btn-primary class to adjust with your primary color